charts/controller/templates/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Values.ingress.enabled - }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "controller.fullname" . }}
+  labels:
+    app: {{ template "controller.name" . }}
+    chart: {{ template "controller.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+{{ toYaml .Values.ingress.annotations | trim | indent 4 }}
+spec:
+  rules:
+    - host: {{ .Values.ingress.host }}
+      http:
+        paths:
+          - path: {{ .Values.ingress.path }}
+            backend:
+              serviceName: {{ template "controller.fullname" . }}
+              servicePort: {{ .Values.ingress.servicePort }}
+{{ end - }}
```

Comment:
`{{ end -}}`
the - should be next to the closing brackets. Currently results in a parse error of the template